src/main/java/org/chainoptim/desktop/features/supplier/controller/SupplierOrdersController.java

```diff
@@ -1,11 +1,153 @@
 package org.chainoptim.desktop.features.supplier.controller;
 
+import com.google.inject.Inject;
+import javafx.application.Platform;
+import javafx.fxml.FXML;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.cell.PropertyValueFactory;
+import javafx.scene.control.cell.TextFieldTableCell;
+import javafx.scene.layout.StackPane;
+import javafx.util.Callback;
+import javafx.util.converter.FloatStringConverter;
+import javafx.util.converter.IntegerStringConverter;
+import org.chainoptim.desktop.features.client.model.ClientOrder;
```

Comment:
Ensure Intellij places org.chainoptim.desktop dependencies before external ones for consistency. Go to:
Settings > Editor > Code Style > Java > Imports 
and under Layout static imports separately add org.chainoptim.desktop and move it to the top.